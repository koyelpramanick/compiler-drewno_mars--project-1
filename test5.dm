""
"&!88"  5 7
    /
"use \n to denote a newline character"
"use \" to  for a quote and \\ for a backslash"
_test
214748364799
@
