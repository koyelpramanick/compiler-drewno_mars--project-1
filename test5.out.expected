STRINGLITERAL:"" [1,1]
STRINGLITERAL:"&!88" [2,1]
INTLITERAL:5 [2,9]
INTLITERAL:7 [2,11]
SLASH [3,5]
STRINGLITERAL:"use \n to denote a newline character" [4,1]
STRINGLITERAL:"use \" to  for a quote and \\ for a backslash" [5,1]
ID:_test [6,1]
INTLITERAL:0 [7,1]
EOF [9,1]
